dask/dataframe/tests/test_multi.py

```diff
@@ -847,6 +847,24 @@ def test_merge(how, shuffle):
     #         pd.merge(A, B, left_index=True, right_on='y'))
 
 
+def test_merge_how_raises():
+    left = pd.DataFrame(
+        {
+            "A": ["A0", "A1", "A2", "A3"],
+            "B": ["B0", "B1", "B2", "B3"],
+        }
+    )
+    right = pd.DataFrame(
+        {
+            "A": ["C0", "C1", "C2", "C3"],
+            "B": ["D0", "D1", "D2", "D3"],
+        }
+    )
+
+    with pytest.raises(Exception):
```

Comment:
nitpick, but you can use a specific class of `Exception` and `match=` to make sure that you are getting the error you expect.

Reply:
+1. Here's an example of another place in the codebase were we specify the specific type of error and message we expect to be raised:

https://github.com/dask/dask/blob/1797c2bbb7c9dcf597840810ed6986f871de333d/dask/array/tests/test_stats.py#L146-L147

Reply:
Thanks for the feedback and example. If the match sentence is too long I can cut it to just `match="how='cross"`. I wasn't sure what do we prefer in this case.